etc/migration/9.2-to-10.0/migration.cql

```diff
@@ -0,0 +1 @@
+ALTER "Principals" ADD "guest" boolean;
```

Comment:
Right, but imagine that the GT tenant is private. When he invites a user with a *.gatech.edu address, we can just invite that user to the private GT tenant. However, when he invites a *.cam.ac.uk address, we probably don't want to invite someone to the Cambridge tenant, as that would start to violate the tenant interaction rules. Inviting him to the GT tenant doesn't really make sense either.

I guess what I'm trying to say is that it seems to make most sense to disallow guest user invites for private tenancies, other than email addresses that match with the private tenant's email domain configuration.